setenv.sh

```diff
@@ -1 +1,2 @@
-export CATALINA_OPTS="$CATALINA_OPTS -javaagent:./dd-java-agent.jar"
\ No newline at end of file
+export CATALINA_OPTS="$CATALINA_OPTS -javaagent:./dd-java-agent.jar"
+export CLASSPATH="$CLASSPATH:/usr/local/tomcat/bin/ecs-logging-core-1.6.0.jar:/usr/local/tomcat/bin/jul-ecs-formatter-1.6.0.jar"
```

Comment:
Is your intention to _append_ or _prepend_ to the classpath? Here you're _appending_.

Also, consider adding a comment